### PR TITLE
Support for MapR 6.0.1

### PR DESCRIPTION
**Pull request by kirtiv1**

### CODE REVIEW

File: images/primary_node/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ RUN . /tmp/${REPO_URL_INSERT_SCRIPT} ${MAPR_VERSION} ${MEP_VERSION}
 RUN rpm --import http://package.mapr.com/releases/pub/maprgpg.key
 
 RUN yum -y install java-1.8.0-openjdk-devel \
+    openssl \
```
Comment (dimaspivak) [inline images/primary_node/Dockerfile]: Nit: can we put this at the end of the list to keep it alphabetically ordered?

File: start.py [hunk 2]
```diff
@@ -83,8 +84,9 @@ def main(args):
         for node in cluster.nodes:
             node.volumes.extend(volumes)
 
-    # MapR 6.0.0 uses CentOS 7 which  needs following settings.
-    if args.mapr_version.startswith('6.0.0'):
+    # MapR versions 6.0.0 onwards use CentOS 7 which needs following settings.
+    mapr_version_tuple = tuple([int(i) for i in args.mapr_version.split('.')])
```
Comment (dimaspivak) [inline start.py]: `[` and `]` not needed...

**Review by dimaspivak — CHANGES_REQUESTED** [review]

File: images/primary_node/Dockerfile (hunk 1 above)
Comment (kirtiv1) [reply images/primary_node/Dockerfile]: Done.

File: start.py (hunk 2 above)
Comment (kirtiv1) [reply start.py]: Done.

**Review by dimaspivak — APPROVED** [review]

**Pull request merged**